atlasdb-feign/src/test/java/com/palantir/atlasdb/http/FeignOkHttpClientsTest.java

```diff
@@ -27,7 +27,7 @@
     @Test
     public void clientDoesNotRetryAtTheOkHttpLevel() {
         OkHttpClient okHttpClient = FeignOkHttpClients.newRawOkHttpClient(Optional.empty(),
-                Optional.empty(), "userAgent");
+                Optional.empty(), "userAgent", true);
         assertThat(okHttpClient.retryOnConnectionFailure()).isFalse();
```

Comment:
seems like a pointless test :S

Reply:
I guess it's really important the default doesn't change?